dist/styles/dark.css

```diff
@@ -18,7 +18,7 @@
         box-shadow: #0008 0px 5px 40px;
     }
     .default-card div.bg {
-        background-color: #000b;
+        background-color: #444b;
```

Comment:
Why is this changed?

Reply:
This is how Fluent Reader's Card look:

![](https://user-images.githubusercontent.com/47271684/90629217-90ee8300-e251-11ea-9d19-8341bbfe9998.png)

This is a card-like component in XAML Controls Gallery:

![](https://user-images.githubusercontent.com/47271684/90629260-a5cb1680-e251-11ea-866c-054aaa5af1d9.png)

The card-like component in UWP app's background is lighter than the app's background, but Fluent Reader's article card is darker. This change made article card's background lighter.